config/v1/types_infrastructure.go

```diff
@@ -1104,6 +1134,13 @@ type IBMCloudPlatformStatus struct {
 	// DNSInstanceCRN is the CRN of the DNS Services instance managing the DNS zone
 	// for the cluster's base domain
 	DNSInstanceCRN string `json:"dnsInstanceCRN,omitempty"`
+
+        // ServiceEndpoints is a list of custom endpoints which will override the default
+        // service endpoints of an IBM Cloud service.
```

Comment:
Something like this sound better?
```
        // ServiceEndpoints is a list of custom endpoints which will override the default
        // service endpoints of an IBM Cloud service. These endpoints are populated from the
        // IBMCloudPlatformSpec, after validation by the Cluster Config Operator, and are
        // consumed by components within the cluster to reach the respective IBM Cloud Services.
```

Reply:
Yes, except, I expect we will need to validate them via the cloud-controller-manager-operator instead of the config operator

Reply:
Does it make sense, to add the support of this (Spec) as followup feature, likely on a 4.x (minor version) boundary?

Say, the Spec and CCCMO validation would be added in 4.15 instead. Meaning 4.14 clusters would allow ServiceEndpoints to be overridden, but only during cluster creation (via Status set by installer), and not post-cluster-creation (via Spec and CCCMO), until I add that support later.

Reply:
I've removed the support for Spec, to allow post-clsuter-creation changes to endpoints, and will see about adding that in 4.15 (or 4.14 if time permits), using validation by CCCMO.